src/clue.js

```diff
@@ -2,27 +2,147 @@
+    lastName: 'Plum',
+    occupation: 'Designer',
+    age: 22,
+    description: 'Billionaire video game designer',
+    image: 'https://66.media.tumblr.com/ee7155882178f73b3781603f0908617c/tumblr_phhxc7EhPJ1w5fh03_540.jpg',
+    color: 'purple'},
+  {
+    firstName: 'Kasandra',
+    lastName: 'Scarlet',
+    occupation: 'Actor',
+    age: 31,
+    description: 'She is an A-list movie star with a dark past',
+    image: 'https://www.radiotimes.com/uploads/images/Original/111967.jpg',
+    color: 'red'},
+  {
+    firstName: 'Eleanor',
+    lastName: 'Peacock',
+    occupation: 'Socialité',
+    age: 36,
+    description: 'She is from a wealthy family and uses her status and money to earn popularity',
+    image: 'https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg',
+    color: 'blue'},
+  {
+    firstName: 'Jack',
+    lastName: 'Mustard',
+    occupation: 'Retired Football player',
+    age: 62,
+    description: 'He is a former football player who tries to get by on his former glory',
+    image: 'https://static.independent.co.uk/s3fs-public/thumbnails/image/2016/07/04/08/unspecified-3.jpg',
+    color: 'yellow'}
+];
+
 
 // Rooms Array
 
-const roomsArray = [];
+const roomsArray = [
+  {name: 'Dining Room'},
+  {name: 'Conservatory'},
+  {name: 'Kitchen'},
+  {name: 'Study'},
+  {name: 'Library'},
+  {name: 'Billiard Room'},
+  {name: 'Lounge'},
+  {name: 'Ballroom'},
+  {name: 'Hall'},
+  {name: 'Spa'},
+  {name: 'Living Room'},
+  {name: 'Observatory'},
+  {name: 'Theater'},
+  {name: 'Guest House'},
+  {name: 'Patio'},
+];
 
 // Weapons Array
 
-const weaponsArray = [];
+const weaponsArray = [
+  {name: 'rope', weight: 10},
+  {name: 'knife', weight: 8},
+  {name: 'candlestick', weight: 2},
+  {name: 'dumbbell', weight: 30},
+  {name: 'poison', weight: 2},
+  {name: 'axe', weight: 15},
+  {name: 'bat', weight: 13},
+  {name: 'trophy', weight: 25},
+  {name: 'pistol', weight: 20},
+  ];
 
 
 // ITERATION 2
 
-function selectRandom() {}
+// Random Selector Suspect
+
+function selectRandomSuspect(suspectsArray) {
+  let randomSuspect = suspectsArray[Math.floor(Math.random() * suspectsArray.length)];
+  return randomSuspect;
+}
+
+// result1 = selectRandom();
+// console.log("Suspect: " + result1.firstName + result1.lastName);
+
+// Random Selector Weapon
+
+function selectRandomWeapon(weaponsArray) {
+  let randomWeapon = weaponsArray[Math.floor(Math.random() * weaponsArray.length)];
+  return randomWeapon
+}
+
+// result2 = selectRandom();
+// console.log("Weapon: " + result2.name);
 
-function pickMystery() {}
+// Random Selector Room
+
+function selectRandomRoom(roomsArray) {
+  let randomRoom = roomsArray[Math.floor(Math.random() * roomsArray.length)];
+  return randomRoom
+}
+
+// result3 = selectRandom();
+// console.log("Room: " + result3.name);
+
+
+function pickMystery(result1, result2, result3) {
+  let mistery = {};
+  mistery.push(result1);
+  mistery.push(result2);
+  mistery.push(result3);
+  return mistery;
+}
```

Comment:
push ist eine Array Methode. 
Du kannst in dieser Aufgabe die erste Funktion benutzen und gibst ein Objekt zurück, z.B. so:
function pickMystery() {

  let mysteryObject = {
    suspect: selectRandom(suspectsArray), //gibt dir ein "random suspect"
    weapon: selectRandom(weaponsArray), // gibt dir ein "random weapon"
    room: selectRandom(roomsArray), // gibt dir ein "random room"
  }
  return mysteryObject; // zufällig zusammengestelltes Mystery(Objekt) zurückgeben
}